sdk/Cargo.toml

```diff
@@ -105,8 +104,8 @@ config = { version = "0.14.0", default-features = false, features = [
 const-hex = "1.14"
 const-oid = { version = "0.9.6", optional = true }
 coset = "0.4.0"
+der = { version = "0.7.9", features = ["alloc", "derive"] }
```

Comment:
Do we use the `derive` functionality in this PR?

Reply:
No. It's been removed. Thanks